src/pspm_gaze_pp.m

```diff
@@ -149,7 +149,7 @@
     preprocessed_gaze.header.sr = gaze_og{1}.header.sr;
   end
   preprocessed_gaze.data = transpose(mean(transpose([preprocessed_gaze.data, preprocessed_gaze_combine.data]),'omitnan'));
-  preprocessed_gaze.header.chantype = pspm_update_channeltype(gaze_og{1}.header.chantype,{'pp',settings.lateral.char.c});
+  preprocessed_gaze.header.chantype = pspm_update_channeltype(gaze_og{1}.header.chantype,settings.lateral.char.c);
```

Comment:
My understanding from the explanation is, the output of `pspm_gaze_pp` should no longer include `pp` in the channel type. In this case, should this be directly `preprocessed_gaze.header.chantype = gaze_og{1}.header.chantype,settings.lateral.char.c;`?

Reply:
Sorry, I seem to understand now. This is to update the chantype from `l` or `r` to `c`. My apologise.